### ISSUE TITLE
[Enhancement]: Resource: aws_cloudwatch_log_metric_filter

### ISSUE THREAD

**Issue by jasuhail** [hashicorp/terraform-provider-aws#29326]
### Description

Would like to metric_transformation_name as output id from the Resource: aws_cloudwatch_log_metric_filter. 

### Affected Resource(s) and/or Data Source(s)

Resource: aws_cloudwatch_log_metric_filter. 

Would like to get output atrribute from this metric filter for metric transformation name

### Would you like to implement a fix?

None

**Comment by bennylu2** [comment]
This is already output in the resource

You can access using `metric_transformation.name` attribute

**Comment by maintux** [comment]
@bennylu2 Nope, the metric_transformation is a block, this means that its type is a list. Moreover the metric_transformation content is converted into a map of strings. So the the right syntax is `metric_transformation[0]["name"]`

**Comment by wmhartl** [comment]
I had to use `.metric_transformation[0].name`
